### DISCUSSION TITLE
BERTopic Probability Score

### DISCUSSION THREAD

**Discussion by rizkiamandaputri** [MaartenGr/BERTopic#1699] [Q&A]
Dear Maarten

I wanna ask you about BERTopic probability score
Can BERTopic understand Bahasa Indonesia text?
So, I tried using Bahasa Indonesia text to measure the probability scores produced by BERTopic. But why were the probability results less accurate?

Thank you Maarten!

**Comment by MaartenGr** [comment]
> Can BERTopic understand Bahasa Indonesia text?

That should be possible as long as you choose a multilingual embedding model and a tokenizer that works with that language.

> So, I tried using Bahasa Indonesia text to measure the probability scores produced by BERTopic. But why were the probability results less accurate?

Could you explain this a bit more? It is not clear what you mean by "But why were the probability results less accurate?". Make sure to give the full code and describe your issue in detail.
